modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,55 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads may use the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning the CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. Use the following table see where process' CPUs run after tuning the node using the Performance Addon Operator:
```

Comment:
First sentence sounds passive with 'would impact'. I would also hyphenate 'latency sensitive'.
So rewrite as '... in a way that impacts latency-sensitive processes...'
Fourth sentence is missing 'see'.
So rewrite as 'Use the following table to see where...'

Reply:
Updated as suggested.